xform-marc21-to-xml/pom.xml

```diff
@@ -125,11 +125,12 @@
         <!-- Symphony is running Oracle 11.2 -->
         <!-- http://www.oracle.com/technetwork/apps-tech/jdbc-112010-090769.html -->
         <dependency>
-            <groupId>com.oracle.jdbc</groupId>
-            <artifactId>ojdbc6</artifactId>
-            <version>12.1.0.2</version>
+            <groupId>com.oracle.weblogic</groupId>
+            <artifactId>ojdbc7</artifactId>
+            <version>12.1.3-0-0</version>
```

Comment:
- Is this client compatible with Symphony?
- Do we have to change the artifact namespace to `com.oracle.weblogic`?  Does that have implications for any includes in the java code (there are none in this PR)?
- Did you run any integration tests on your laptop using this branch against our Symphony system?